client/my-sites/stats/controller.jsx

```diff
@@ -176,6 +177,12 @@ export default {
 
 	site: function( context, next ) {
 		const { params: { site_id: givenSiteId }, query: queryOptions, store } = context;
+
+		if ( 'simplePaymentsEmailTour' === get( queryOptions, 'tour' ) && ! isDesktop() ) {
+			window.location.href = 'https://en.support.wordpress.com/simple-payments/';
```

Comment:
How difficult would it be to add a tracks event here? I think folks would be interested to know how many folks open this on mobile. 

If we have ga setup on en.support.wordpress, alternatively we can add a query param to mark this.

cc @jancavan @anneforbush

Reply:
> I think folks would be interested to know how many folks open this on mobile.

👍 If we're sending this out via MailChimp, I think their reports can provide a desktop vs mobile breakdown.

Reply:
For the call to action link in the email?

Reply:
It's just for the email campaign itself though, but I was thinking it may be similar info in terms of mobile vs desktop (ie user loads campaign on their phone which MailChimp records, clicks on cta, page with Tracks event then records this as a mobile user). We should still use Tracks though; just pointed that out as supplementary data if needed.

Reply:
Let's add the tracks event here then @Copons  and we can try handling the race condition in another PR. I believe we're run into this case where we'd like to force stats to record right before navigating away from wp.com